app/konnect/analytics/use-cases/latency.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Diagnosing Latency Issues
+content_type: tutorial
+book: use-cases
+chapter: 2
+---
+
+Custom reporting in {{site.konnect_saas}} gives you the power to monitor your API data in detail and export that data to CSV. 
+Let's go through some example situations where you could leverage custom reports.
+
+In this scenario, Ryan is a platform owner who supervises the implementation and maintenance of the API Gateway infrastructure that governs all APIs at ACME. A few minutes ago, he received an email that stated that the response latency seemed to be very high and and causing a bad customer experience. To better understand the source of the delays, Ryan decides to create two reports to compare the **average upstream** and **Kong latency** for all traffic to the production servers in the last hour.
```

Comment:
> To better understand the source of the delays, Ryan decides to create two reports to compare the **average upstream** and **Kong latency** for all traffic to the production servers in the last hour.

Could be me, but I wasn't sure how Ryan decided that those were the two  metrics he wanted to measure based on his scenario.

Reply:
Also, nitpick, but for the name, could we use one from Google's list in the style guide? https://developers.google.com/style/examples#example-person-names

Reply:
Kong and Upstream latency are the most important to determine where a performance bottleneck might be. Kong allows you to understand the Kong Gateway latency where as Upstream is the actual API that Kong proxies. It's very well known as a platform admin to which metrics you need to look into to understand both.